Game/Character/CharacterFunctionality.h

```diff
@@ -29,6 +31,7 @@ class CharacterFunctionality
 	: public Functionality<
 		CharacterFunctionality,
 		Policy::Set<
+			Policy::Add<InputSystem, Policy::EPermission::Write>,
```

Comment:
Why does CharacterFunctionality need InputSystem access? I might be missing something

Reply:
Will remove

Reply:
Never mind, it's used by WeaponSwingHitShapeAction.